src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/attribute_editor.cljs

```diff
@@ -105,9 +105,29 @@
              :else {:success typed})))
    :render
    (fn [{:keys [props state after-update]}]
-     (let [{:keys [editing?]} props]
+     (let [{:keys [editing? writer?]} props]
        [:div {}
-        (style/create-section-header "Workspace Attributes")
+        (style/create-section-header
+          [:div {} "Workspace Attributes"
+           (style/create-inline
+             [:a {:style {:textDecoration "none" :margin "7px 0 0 1em"}
+                  :href (str (config/api-url-root) "/cookie-authed/workspaces/"
+                          (:namespace (:workspace-id props)) "/"
+                          (:name (:workspace-id props)) "/exportAttributesTSV")
+                  :onClick  #(utils/set-access-token-cookie (utils/get-access-token))
+                  :target "_blank"}
```

Comment:
This link is inconsistent with other download links.  Compare to the bucket link on this page.  I think we don't want `:textDecoration "none"`.